OWNERS_ALIASES

```diff
@@ -122,14 +122,15 @@ aliases:
     - jimangel
     - kbarnard10
     - mistyhacks
+    - rajakavitha1
     - ryanmcginnis
     - steveperry-53
     - stewart-yu
     - tengqm
     - tfogo
     - zacharysarah
     - zhangxiaoyu-zidif
-    - zparnold
+    - zparnold    
```

Comment:
Please configure your editor not to add or remove trailing whitespace.

Reply:
okay! thank you :)